tests/FSharp.Test.Utilities/Compiler.fs

```diff
@@ -1618,7 +1664,7 @@ Actual:
 
             let expectedErrors = expected |> List.map (fun error -> errorMessage error)
             let expectedErrorsAsStr = expectedErrors |> String.concat ";\n" |> sprintf "[%s]"
-            let sourceErrors = source |> List.map (fun error -> errorMessage { error with Range = adjustRange error.Range libAdjust })
+            let sourceErrors = source |> List.distinctBy (fun ei -> ei.Range.StartLine, ei.Range.StartColumn, ei.Range.EndLine, ei.Range.EndColumn, ei.Message) |> List.map (fun error -> errorMessage { error with Range = adjustRange error.Range libAdjust })
```

Comment:
The same here - I would prefer tests to fail if we see duplicates, and indicate there is a bug somewhere.